model/state_model_nlparam.jl

```diff
@@ -75,14 +77,14 @@ y_check = Dict()
 
 
 #subsets for model equation controls
-sub_set_y = filter(x -> y_check[x] != 0.0, combvec(regions, sectors));
-sub_set_a = filter(x -> a_set[x[1], x[2]] != 0.0, combvec(regions, goods));
-sub_set_x = filter(x -> get(sld[:s0], (x[1], x[2]), 0.0) != 0.0, combvec(regions, goods));
-sub_set_pa = filter(x -> get(sld[:a0], (x[1], x[2]), 0.0) != 0.0, combvec(regions, goods));
-sub_set_pd = filter(x -> get(sld[:xd0], (x[1], x[2]), 0.0) != 0.0, combvec(regions, goods));
-sub_set_pk = filter(x -> get(sld[:kd0], (x[1], x[2]), 0.0) != 0.0, combvec(regions, goods));
-sub_set_py = filter(x -> get(sld[:kd0], (x[1], x[2]), 0.0) != 0.0, combvec(regions, goods));
-sub_set_py = filter(x -> y_check[x[1], x[2]] != 0, combvec(regions, goods));
+sub_set_y = filter(x -> y_check[x] != 0.0, permute(regions, sectors));
```

Comment:
`combvec` is really well-written! I extended `permute` (one of the SLiDE built-in functions) to include this, since they do similar things. Now, we can use `permute` instead to stick with SLiDE features and avoid defining this function at the top of the file. Nice job! :)

Reply:
Thanks. ```combvec``` was actually written by Sourabh. Sounds good on ```permute```.